hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -297,6 +297,8 @@ def get_different_seeds_hyperdrive_config(self) -> HyperDriveConfig:
 
 
 class DatasetParams(param.Parameterized):
+    input_datasets: List[str] = param.List(default=[], class_=str, doc="Do not use. For AML SDK v2 only")
```

Comment:
this attribute must have the same name as literal strings that you use elsewhere, is there a test for that?

Reply:
It's the same name as this command line arg that gets passed into the Azure ML (v2) command: 
` --input_datasets=${{inputs.input_datasets}}`

There isn't currently an explicit test for this - I guess what we'd want to do is submit an AML v2 command (passing this arg in) and then within the Job, check that the LightningContainer has the expected value of input_datasets? (and the same as above for output_datasets too).

However we do implicitly test this via the smoke tests since they now use the v2 sdk and they would fail to find the data if input_datasets wasn't set...So maybe we can leave the explicit tests for a future PR?